src/tfr_dumping/src/dumping_action_client.cpp

```diff
@@ -0,0 +1,25 @@
+#include <ros/ros.h>
+#include <tfr_msgs/EmptyAction.h>
+#include <actionlib/client/simple_action_client.h>
+/**
+ *test code never to be seen againb
+ * */
+
+int main(int argc, char** argv)
+{
+    ros::init(argc, argv, "simple_action_client");
+    ros::NodeHandle n;
+    actionlib::SimpleActionClient<tfr_msgs::EmptyAction> client("dump", true);
+    ROS_INFO("client waiting for server");
+    client.waitForServer();
+    ROS_INFO("client connected to server");
+
+    ros::Duration(1).sleep();
```

Comment:
Why are we sleeping here?

Reply:
Probably left over from an old manual test case, I'll delete it.

Reply:
I checked it was just leftover test code.

Reply:
/removed it